README.md

```diff
@@ -133,35 +141,6 @@ By default, `bundle exec rake` will run the unit tests first, and then `rubocop`
 
     RUBOCOP_FIRST=1 bundle exec rake
 
-### CF Acceptance Tests (CATs)
```

Comment:
Maybe this could be generalized to just say "Acceptance Tests" and point folks to this README for more info:
https://github.com/cloudfoundry/cloud_controller_ng/tree/master/spec#testing-the-cloud-controller

Reply:
One thing i had privately imagined is maybe a “contributing” link over to capi-k8s-release but we haven’t that out yet. I kinda think having acceptance testing links without having "development deploy" links is more confusing than helpful.